### DISCUSSION TITLE
release next release

### DISCUSSION THREAD

**Discussion by kherink** [sceneview/sceneview#20] [General]
First of all thanks for taking the time and sharing this project!
I want to ask if there is a release roadmap or at least an estimate on when the next release will become officially available.
Currently we have 0.0.3 but that is quite far behind the unreleased code in this repo.

**Comment by ThomasGorisse** [comment]
Hi,

Thanks for thanking the team.
We are quite near from the 0.1.0.
No Roadmap cause we are not a company.
BUT we try to focus more on users needs than on product specifications.
While waiting for the release, you could try to clone the repo. It's very easy to use the depency with `publishToMaven()` from this project and add `mavenLocal()` to your project root `build.gradle`.

If you or anyone reading this could help us bringing Sceneform last fix commits in here, it would make everyone gain time.

**Comment by ThomasGorisse** [comment]
The latest release is **v4.0.0-rc.1** — a major rewrite with a fully Compose-declarative API. The library has had many releases since this discussion was opened. Check the [releases page](https://github.com/sceneview/sceneview/releases) for the full history.
